### DISCUSSION TITLE
Client id and secret

### DISCUSSION THREAD

**Discussion by gio82c** [jbergler/hass-ttlock#140] [Q&A] [answered]
Hi.

I have changed account and now it only asks me user and password of the ttlock account, if I remember correctly it should also ask me client id and client secret, or am I wrong? The problem is that I do not receive updates on user and last trigger, but I can open the locks... I tried to uninstall and reinstall from HACS but with no results... can you confirm that in any case it only asks for user and password?

**Comment by jbergler** [comment, marked as answer]
Yes, that's expected. Your oauth credentials are saved as a "Application Credentials". If you want to re-enter them you can do so from the context menu on the integrations page.

https://www.home-assistant.io/integrations/application_credentials/

If you're not receiving updates, please make sure the webhook is configured correctly, and if in doubt enable debugging and watch the logs after you lock/unlock the lock to see if the events are pushed to your home assistant by the ttlock cloud.

**Reply by gio82c** [reply]
Thanks
